lib/puppet/provider/cs_location/crm.rb

```diff
@@ -17,8 +17,30 @@
   # Decided to just go with relative.
   commands crm: 'crm'
 
+  # Path to the pacemakerd binary to check version for resource_discovery feature
+  # Decided to just go with relative.
+  commands pacemakerd: 'pacemakerd'
+
   mk_resource_methods
 
+  # rubocop:disable Lint/HandleExceptions
+  # we need to check if we run at least pacemakerd version 1.1.13 before enabling feature discovery
+  # see http://blog.clusterlabs.org/blog/2014/feature-spotlight-controllable-resource-discovery
+  begin
+    pacemakerd_version_string = pacemakerd('--version')
+    if pacemakerd_version_string
+      pacemakerd_version = pacemakerd_version_string.scan(%r{\d+\.\d+\.\d+}).first unless pacemakerd_version_string.nil?
```

Comment:
why is this `unless` necessary?
we already have that `if`
one of them should go

Reply:
You are right. Will change that with the next push. Thanks